turbopack/crates/turbo-tasks/src/vc/mod.rs

```diff
@@ -200,14 +201,11 @@ where
     }
 }
 
-impl<T> Copy for Vc<T> where T: ?Sized + Send {}
-
-unsafe impl<T> Send for Vc<T> where T: ?Sized + Send {}
-unsafe impl<T> Sync for Vc<T> where T: ?Sized + Send {}
```

Comment:
These unsafe impls were okay (there's no way to construct a `Vc<T>` that isn't `Send + Sync`), they weren't providing hardly any value because `Send + Sync` is auto-implemented already when `T: Send + Sync`.